frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -135,56 +137,66 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
         <Trans t={t} ns="ceph-storage-plugin" values={{ name }}>
           Adding capacity for <strong>{{ name }}</strong>, may increase your expenses.
         </Trans>
-        <div className="ceph-add-capacity__modal">
-          <div
-            className={classNames('ceph-add-capacity__sc-dropdown', {
-              'ceph-add-capacity__sc-dropdown--margin': !isNoProvionerSC,
-            })}
-          >
-            <label className="control-label" htmlFor="storageClass">
-              {t('ceph-storage-plugin~Storage Class')}
-              <FieldLevelHelp>{storageClassTooltip(t)}</FieldLevelHelp>
-            </label>
+        <FormGroup
+          className="pf-u-pt-md pf-u-pb-sm"
+          id="add-cap-sc-dropdown__FormGroup"
+          fieldId="add-cap-sc-dropdown__FormGroup"
+          isRequired
+        >
+          <Title className="pf-u-pb-sm" headingLevel="h5">
+            {t('ceph-storage-plugin~Storage Class')}&nbsp;&nbsp;
+            <Tooltip position="top" content={<>{storageClassTooltip(t)}</>}>
+              <QuestionCircleIcon className="pf-u-secondary-color-100" />
```

Comment:
Form group has a `label` prop which can take a react element as well as a `labelIcon` to support the icon.
Refer this for more help: https://www.patternfly.org/v4/components/form
I would assume we can leverage both of these props here.